packages/longhorn/longhorn.patch

```diff
@@ -0,0 +1,12 @@
+diff -x '*.tgz' -x '*.lock' -uNr packages/longhorn/charts-original/Chart.yaml packages/longhorn/charts/Chart.yaml
+--- packages/longhorn/charts-original/Chart.yaml
++++ packages/longhorn/charts/Chart.yaml
+@@ -25,3 +25,8 @@
+ - https://github.com/longhorn/longhorn-ui
+ - https://github.com/longhorn/longhorn-tests
+ version: 1.0.1
+++annotations:
++  catalog.rancher.io/certified: rancher
++  catalog.rancher.io/experimental: true
++  catalog.rancher.io/namespace: longhorn-system
++  catalog.rancher.io/release-name: longhorn
```

Comment:
Assuming you'll want a custom UI form to launch longhorn instead of relying only on questions.yml, you will need to add in the 
```
catalog.cattle.io/ui-component:
```

and create an issue in github.com/rancher/dashboard.

Reply:
I think we can deal with this part later since we don't have custom UI form now?

Reply:
If you plan on leveraging questions.yml, then you should add that file in the overlay folder. Otherwise you will not have any UI and only ability to edit yaml.

Reply:
I see. Is there any way to test this?

Reply:
In fact we should add 
```
catalog.cattle.io/ui-component: longhorn
```

See https://rancher.atlassian.net/wiki/spaces/EN/pages/563609933/Helm+Charts+Requirements

Reply:
Let's skip questions.yml for now, since we will have the UI with Rancher.

If we need it later, we can always add that.

We still need to add `catalog.cattle.io/ui-component: longhorn` label.